src/renderer/entities/price/model/currency-model.ts

```diff
@@ -0,0 +1,84 @@
+import { createEvent, createStore, createEffect, forward, sample } from 'effector';
+
+import { kernelModel } from '@renderer/shared/core';
+import { CurrencyConfig, fiatService } from '@renderer/shared/api/price-provider';
+import { DEFAULT_CURRENCY_CODE, DEFAULT_CURRENCY_CONFIG } from '../lib/constants';
+
+export const $currencyConfig = createStore<CurrencyConfig[]>([]);
```

Comment:
are you sure about naming? `$currencyConfig` variable and `CurrencyConfig` type are confused, because first is array but second is object.